docs/pages/blog/react-19-update.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Migrating MUI X to React 19
```

Comment:
I think we should be more descriptive with the title to make it clear right away that this is about _internal_ migration (rather than a guide for users on upgrading their MUI app to React 19). We could borrow from the description:

```suggestion
title: How we migrated MUI X to React 19
```